code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/elite.dm

```diff
@@ -323,7 +325,7 @@ obj/structure/elite_tumor/proc/onEliteWon()
 	smooth = SMOOTH_TRUE
 	layer = BELOW_MOB_LAYER
 	var/mob/living/carbon/human/activator = null
-	var/mob/living/simple_animal/hostile/asteroid/elite/ourelite = null
+	var/mob/living/ourelite = null
```

Comment:
What's this about

Reply:
Leftover accident from some testing, though keeping doesn't do anything.  I'll change it back.